utilities/qneg.meta

```diff
@@ -61,7 +61,7 @@
   intent = in
 [ q ]
   standard_name = ccpp_constituents
-  units = kg kg-1
+  units = none
```

Comment:
I was told not that long ago (I think) that I should be using  kg kg-1 for ccpp_constituents - that's what I'm doing for the zm code.

Reply:
I think it needs to be none. That's what's in the dictionary and the units are (at least currently) determined by the constituent properties array

Reply:
OK - I'll change my ZM metadata files as well.